src/cromshell/log.py

```diff
@@ -6,7 +6,7 @@
 def configure_logging(verbosity):
     """Set up logging for the cromshell module"""
 
-    import cromshell# pylint: disable=C0415
+    import cromshell  # pylint: disable=C0415
```

Comment:
Why does this import happen here instead of at the top?

Reply:
Add comment that having the cromshell import here instead at the top is a speed optimization

Reply:
done